tests/frame/to_numpy_test.py

```diff
@@ -0,0 +1,21 @@
+from __future__ import annotations
+
+from typing import Any
+
+import numpy as np
+
+import narwhals.stable.v1 as nw
+
+
+def test_convert_numpy(constructor: Any) -> None:
+    data = {"a": [1, 3, 2], "b": [4, 4, 6], "z": [7.1, 8, 9]}
+    df_raw = constructor(data)
+    result = nw.from_native(df_raw, eager_only=True).to_numpy()
+
+    expected = np.array([[1, 3, 2], [4, 4, 6], [7.1, 8, 9]]).T
+    np.testing.assert_array_equal(result, expected)
```

Comment:
`pandas_nullable` and `pandas_pyarrow` will convert `7.0` to integer, therefore the type check fails. Using `7.1` fixes that